docs/examples/sentiment_analysis/index.rst

```diff
@@ -4,16 +4,24 @@ Sentiment Analysis
 .. container:: cards
 
    .. card::
-      :title: Sentiment Analysis by Fine-tuning Word Language Model
+      :title: Fine-tuning LSTM-based Language Model
       :link: sentiment_analysis.html
 
       See how to fine-tune a pre-trained language model to perform sentiment analysis on movie reviews.
 
+   .. card::
+      :title: Training Structured Self-attentive Sentence Embedding
+      :link: ../sentence_embedding/self_attentive_sentence_embedding.html
+
+      See how to use GluonNLP to build more advanced model structure for extracting sentence
+      embeddings to predict Yelp review rating.
+
+
 
 .. toctree::
    :hidden:
    :maxdepth: 2
 
    sentiment_analysis.ipynb
-
+   ../sentence_embedding/self_attentive_sentence_embedding.ipynb
```

Comment:
What about moving self_attentive_sentence_embedding.ipynb to the sentiment analysis folder?